swerve_pico/control-library/control-library.cpp

```diff
@@ -0,0 +1,36 @@
+
+#include "pid.cpp"
```

Comment:
you should never include .c or .cpp files, instead, you use header files. in this case, you'd want to put the class definition with just the function signatures (ex. `void Setup();`) in a file called `control-library.h`, then in the corresponding cpp file, you define each function like this:
```
SwerveDrive::Setup() {
    //code
}
```

and so on, not organized/indented into a class structure. If you want an example, look at [this .h file](https://github.com/odriverobotics/ODriveArduino/blob/master/src/ODriveUART.h) paired with [this .cpp file](https://github.com/odriverobotics/ODriveArduino/blob/master/src/ODriveUART.cpp)